src/protocols/displayport.stanza

```diff
@@ -54,16 +57,38 @@ doc: \<DOC>
 @brief DisplayPort Bundle
 The Data Bundle consists of 4 lane pairs (1 RX or TX pair set = 1 lane)
 The bundle also constrains an AUX channel (differential pair) and a HPD pin
+
+@member aux Differential pair for the auxiliary channel
+@member lane Four differential pairs for data transfer
+@member hpd Hot plug detect
 <DOC>
 
 public pcb-bundle displayport :
   name = "DisplayPort"
   description = "DisplayPort Digital Display Interface"
   port aux  : diff-pair
-  port lane : diff-pair[4]
+  port lane : diff-pair[DISPLAYPORT_NUM_LANES]
   port hpd  : pin ; hot plug detect
```

Comment:
Is this part of the standard high speed connections for the transport bundle - or should this be in the connector bundle ? 

Is it typical for SoC peripherals for Display port to have an explicit pin for the HPD ? Or is it usually just a GPIO ? 

If it is just a GPIO - then we should put it in the connector bundle.

Reply:
I remember seeing this on ICs so I kept it here. I think that it might be used by the interface for sync?